components/blog-index/server.css

```diff
@@ -113,7 +117,7 @@
     display: flex;
 
     width: auto;
-    height: 200px;
+    max-height: 200px;
```

Comment:
Would be good if @pepelsbey can take a look, as this might re-introduce layout shift?